src/checkin/model/session/Session.ts

```diff
@@ -7,4 +7,5 @@ export interface Session {
   userId: string
   data: SessionData
   userInformation: UserInformation
+  extra_passenger_information?: Array<{ name: string, passport: string }>
```

Comment:
Instead of defining the type as Array<{ name: string, passport: string }> it would be more optimal to represent the model by creating an interface (e.g., ExtraPassengerInfo) and then defining the field as extra_passenger_information?: ExtraPassengerInfo[]. This approach proves beneficial, especially considering the consistent use of Array<{ name: string, passport: string }> to represent the data type in five other sections of the code.

Reply:
Thaks for the suggestion, i will add it right away!